bootstrap-salt.sh

```diff
@@ -6750,6 +6750,12 @@ install_macosx_stable_deps() {
 install_macosx_git_deps() {
     install_macosx_stable_deps || return 1
 
+    echo "$PATH" | grep -q /usr/local/bin
+    if [  $? -eq 1 ]; then
```

Comment:
How about to include the whole command into the `if` statement expression?
No need for additional return code check:
```suggestion
    if ! echo "$PATH" | grep -q /usr/local/bin; then
```